### PR TITLE
fix: bump to fedora:38 in dockerized tests, because 34 is missing, bump Multus, OVS, and kubevirtci

### LINKED ISSUE #267: Dockerized tests fail due to missing Fedora 34 image
The fedora:34 image no longer exists on `quay.io`, thus dockerized tests are failing.

```bash
make docker-test                                                                              
hack/test-dockerized.sh
[+] Building 0.4s (3/3) FINISHED                                                                                   
 => [internal] load build definition from Dockerfile                                                          0.0s
 => => transferring dockerfile: 999B                                                                          0.0s
 => [internal] load .dockerignore                                                                             0.0s
 => => transferring context: 2B                                                                               0.0s
 => ERROR [internal] load metadata for quay.io/fedora/fedora:34-x86_64                                        0.3s
------
 > [internal] load metadata for quay.io/fedora/fedora:34-x86_64:
------
failed to solve with frontend dockerfile.v0: failed to create LLB definition: quay.io/fedora/fedora:34-x86_64: not found
make: *** [docker-test] Error 1
```

### PR DESCRIPTION
**Pull request by courtland**
**What this PR does / why we need it**:

quay.io pulled docker images for fedora:34

**Special notes for your reviewer**:

Fixes #267

**Release note**:

```release-note
Fix dockerized tests after Fedora 34 EOL
```

### CODE REVIEW

**Comment by phoracek** [comment]
CI failed installing dependencies on the e2e test cluster:

```
 + ./cluster/cli.sh ssh node01 -- sudo dnf install -y centos-release-nfv-openvswitch
selecting podman as container runtime
CentOS-8 - NFV OpenvSwitch                      8.2 kB/s | 3.0 kB     00:00    
CentOS-8 - NFV OpenvSwitch                      139 kB/s | 236 kB     00:01    
CentOS Stream 8 - AppStream                      12 kB/s | 4.4 kB     00:00    
CentOS Stream 8 - AppStream                      26 MB/s |  29 MB     00:01    
CentOS Stream 8 - BaseOS                        9.6 kB/s | 3.9 kB     00:00    
CentOS Stream 8 - BaseOS                         17 MB/s |  33 MB     00:01    
CentOS Stream 8 - Extras                         10 kB/s | 2.9 kB     00:00    
CentOS Stream 8 - Extras common packages         16 kB/s | 3.0 kB     00:00    
CentOS Stream 8 - Extras common packages         13 kB/s | 6.4 kB     00:00    
Stable Releases of Upstream github.com/containe  64 kB/s | 1.7 kB     00:00    
Stable Releases of Upstream github.com/containe 623 kB/s |  66 kB     00:00    
devel:kubic:libcontainers:stable:cri-o:1.22 (Ce 1.6 kB/s | 248  B     00:00    
Errors during downloading metadata for repository 'devel_kubic_libcontainers_stable_cri-o_1.22':
  - Status code: 404 for https://storage.googleapis.com/kubevirtci-crio-mirror/devel_kubic_libcontainers_stable_cri-o_1.22/repodata/repomd.xml (IP: 142.250.138.128)
Error: Failed to download metadata for repo 'devel_kubic_libcontainers_stable_cri-o_1.22': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
```

Trying again, in case it was a temporary issue. If not, we will probably need to fix the CI environment.

**Comment by phoracek** [comment]
Checking if bump of kubevirtci and OVS helps: https://github.com/k8snetworkplumbingwg/ovs-cni/pull/270

**Comment by phoracek** [comment]
/retest

Giving last chance to the flaky test.

**Comment by phoracek** [comment]
/lgtm
/approve

Wonderful! Thanks for the fix @courtland

**Comment by phoracek** [comment]
/retest

Sorry about the flaky CI.

**Comment by phoracek** [comment]
Merging manually. The test failure is not related to the PR.

**Comment by courtland** [comment]
> /retest
> 
> Sorry about the flaky CI.

It's all good. What would any CI be without some flakes? :D

Thanks for your help in getting this through.

**Comment by phoracek** [comment]
Thank you for pushing this forward! Much appreciated o/

File: cluster/up.sh [hunk 1]
```diff
@@ -23,15 +23,12 @@ $(cluster::path)/cluster-up/up.sh
 
 echo 'Installing Open vSwitch on nodes'
 for node in $(./cluster/kubectl.sh get nodes --no-headers | awk '{print $1}'); do
-    ./cluster/cli.sh ssh ${node} -- sudo dnf install -y centos-release-nfv-openvswitch
-    ./cluster/cli.sh ssh ${node} -- sudo dnf install -y openvswitch2.16 dpdk
     ./cluster/cli.sh ssh ${node} -- sudo systemctl daemon-reload
+    ./cluster/cli.sh ssh ${node} -- sudo systemctl enable openvswitch
     ./cluster/cli.sh ssh ${node} -- sudo systemctl restart openvswitch
+    ./cluster/cli.sh ssh ${node} -- sudo systemctl restart NetworkManager
```
Comment (oshoval) [inline cluster/up.sh:29]: `sudo systemctl enable --now openvswitch`
should be enough right ?
this what we are doing on CNAO
the cluster is recreated from scratch, so the file system is refreshed and NM is fresh started as well

**Comment by oshoval** [comment]
@phoracek 
note please that if you take this you will need also those maybe
https://github.com/k8snetworkplumbingwg/ovs-cni/pull/281
https://github.com/k8snetworkplumbingwg/ovs-cni/pull/282

they are due to this PR

**Pull request merged**